jfrunit-events-generator/src/test/java/org/moditect/jfrunit/generator/PascalToSnakeCaseTest.java

```diff
@@ -0,0 +1,15 @@
+package org.moditect.jfrunit.generator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+public class PascalToSnakeCaseTest {
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/pascalToSnakeCaseTest.csv", numLinesToSkip = 1)
+    public void eventNameToSnakeCase(String input, String expected) {
```

Comment:
Can we have a test for null input and empty string, too?

Reply:
added
```
,
"",""
```
in csv for null and empty string respectively